test/e2e/batch/batch_test.go

```diff
@@ -251,3 +253,27 @@ func assertRestOk(url string, client tutils.HTTPClient) {
 		panic(fmt.Sprintf("Expected Status Code 200, received %d", rsp.StatusCode))
 	}
 }
+
+func TestBatchPodLabel(t *testing.T) {
```

Comment:
@durgeshanaokar not sure we need to create a separate test for just batch labels. We can embedded this validation in any Batch CR test.
@ryanemerson WDYT?